src/main/java/uk/ac/cam/cl/dtg/segue/comm/EmailManager.java

```diff
@@ -186,6 +186,34 @@ public void sendTemplatedEmailToUser(final RegisteredUserDTO userDTO, final Emai
     this.filterByPreferencesAndAddToQueue(userDTO, emailCommunicationMessage);
   }
 
+  /**
+   * Function that enables PW reset response to be sent to a random email address (not to a known user).
```

Comment:
Are we saying that if someone tries to reset the PW on an email we don't currently have a user for, we send that account an email?! This sounds like a bad idea to me.

Reply:
As per the ticket (https://github.com/isaaccomputerscience/isaac-cs-issues/issues/401) the idea is that if someone tries to reset a PW but the account does not exist it will send an email to notify the email user that a PW reset was attempted for that email. This is the email text:

```
Hello,

We have received a password reset request for this email address for an [Isaac Computer Science](https://isaaccomputerscience.org/) account. 

We cannot send you a password reset link as this email address is not registered. 

If you would like to create an Isaac Computer Science account please fill in the registration form on our Sign Up page: {{registrationURL}} 

If you didn't request this password reset, you can safely ignore this email.

Regards,

{{sig}}
```

Reply:
I would regard that as a potential DoS/spam channel. Effectively we're sending unsolicited emails to an unregistered address. That has potential GDPR issues, but also if someone attempts brute forcing we're potentially sending *lots* of unsolicited emails to an innocent party. To prevent that we'd have to record the email address without consent, and obvs there are GDPR issues with that too.